### ISSUE TITLE
[BUG/ISSUE] LUO_WETDEP doesn't work regardless of build configuration setting

### ISSUE THREAD

**Issue by LiamBindle** [geoschem/GCHP#120]
@msulprizio reported:
> Hi all — Bob and I discovered LUO_WETDEP is not actually activated in GCHP even when passing -DLUO_WETDEP=on. From the CMakeCache.txt, we get:
> ```
> //No help, variable specified on the command line.
> LUO_WETDEP:UNINITIALIZED=on
> ```
> whereas for GCClassic, we get:
> ```
> //Switch to build the Luo et al (2019) wetdep scheme into GEOS-Chem
> LUO_WETDEP:BOOL=on
> ```

**Comment by LiamBindle** [comment]
I'll merge the fix into `patch/13.1.3`. Does this warrant a 13.0.4 patch too?

**Comment by msulprizio** [comment]
I think it should be fine to fix in 13.1.3 for now. Users of GCHP 13.0 that want to try out the older Luo et al. (2019) option can manually implement the fix above.

**Comment by LiamBindle** [comment]
Sounds good, that makes sense. 

For anyone that wants this fix you can run the following commands in the top-level of the GCHP source code:
```console
$ git fetch
$ git cherry-pick b7bc4dc5925b78e184b895e6ae619bff69478b46
```

**Comment by msulprizio** [comment]
With the fix in PR #121, I was still getting:

```
//No help, variable specified on the command line.
LUO_WETDEP:UNINITIALIZED=on
```

I made some further updates to src/GCHP_GridComp/GEOSChem_GridComp/CMakeLists.txt and submitted a new PR #122. I now get:

```
//Switch to build the Luo et al (2020) wetdep scheme into GEOS-Chem
LUO_WETDEP:BOOL=on
```

@LiamBindle me know if you have any concerns before we merge this into `dev`.
